qacode/core/enums/enum_base.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""package qacode.core.enums.enum_base"""
+
+
+from enum import Enum
+
+
+class EnumBase(Enum):
+    """Enum Class with base methods to be used
+        at inherit classes
+    """
+
+    @classmethod
+    def get_properties(cls):
+        """Return enum values"""
+        return [item.value for item in type(cls)]
+
+    @classmethod

```

Comment:
While not really implemented, it's more idiomatic and fastest to try to get the enum from the constructor class using the value:

```python
from enum import Enum

class MyEnum(Enum):
    A = 1
    B = 2

MyEnum(1)  # Returns MyEnum.A
MyEnum(3)  # Raises ValueError
```

At a minimum, I would rewrite the `has_property` to use the class constructor instead to iterate over the values.

Reply:
will be resolved at #181